README.md

```diff
@@ -1,19 +1,144 @@
 # MongoosePush
 
-**TODO: Add description**
+[![Build Status](https://travis-ci.org/esl/MongoosePush.svg?branch=initial_implementation)](https://travis-ci.org/esl/MongoosePush) [![Coverage Status](https://coveralls.io/repos/github/esl/MongoosePush/badge.svg?branch=initial_implementation)](https://coveralls.io/github/esl/MongoosePush?branch=initial_implementation)
 
-## Installation
+**MongoosePush** is simple (seriously) **REST** service written in **Elixir** providing ability to **send push
+notification** to `FCM` (Firebase Cloud Messaging) and/or
+`APNS` (Apple Push Notification Service) via their `HTTP/2` API.
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `mongoose_push` to your list of dependencies in `mix.exs`:
+## Quick start
 
+### Docker
+
+Soon :)
+
+### Local build
+
+#### Perquisites
+
+* Elixir (http://elixir-lang.org/install.html)
+* Rebar3 (just enter ```mix local.rebar```)
+
+#### Build and run
+
+Build step is really easy. Just type in root of the repository:
+```bash
+MIX_ENV=prod mix do certs.dev, deps.get, compile, release
+```
+
+After this step you may try to run the service via:
+```bash
+_build/prod/rel/mongoose_push/bin/mongoose_push console
+```
```

Comment:
Could you elaborate? What do you mean by `support for releases` exactly? This `README.md` shows how to run this project from the release.